aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java

```diff
@@ -567,15 +567,19 @@ private static void compileCallIndirect(MethodVisitor asm, int typeId, FunctionT
         int slot = type.params().stream().mapToInt(AotUtil::slotCount).sum();
 
         // parameters: arguments, funcTableIdx, tableIdx, instance
-        emitBoxArguments(asm, type.params());
+        // emitBoxArguments(asm, type.params());
+        // asm.visitLdcInsn(LONG);
+        for (int i = 0; i < type.params().size(); i++) {
```

Comment:
This needs to use `slotCount()` to compute the slot parameter correctly, as long/double use two slots in JVM bytecode. This annoyance means you always need to know the type you are dealing with.

The incoming parameters to this method are a native Java value: int, long, float, double